pyproject.toml

```diff
@@ -14,13 +14,14 @@ classifiers = [
 include = [
   # Explicitly include non-Python files and gitignored files:
   "LICENSE",
-  "*.m5o",
+  "scripts/build.py",
```

Comment:
@WillDaSilva - Do we need this to be bundled/included into the distribution in order for the build to work? If not, can we exclude without impact to the users?

Reply:
It is included by default now that it resides in the root of the project. It must reside in the root of the project due to a Poetry bug.